.eslintrc.json

```diff
@@ -8,7 +8,16 @@
   ],
   "rules": {
     "linebreak-style": "off",
-    "no-underscore-dangle": "off"
+    "no-underscore-dangle": "off",
+    "react/jsx-filename-extension": [
+      1,
+      {
+        "extensions": [
+          ".js",
+          ".jsx"
+        ]
+      }
+    ]
```

Comment:
This was added in order to remove an ESLint error that didn't allow jsx code in a `.js` file.